### ISSUE TITLE
Array pointing back to itself in an infinite loop

### ISSUE THREAD

**Issue by cristianbuse** [WaynePhillipsEA/twinbasic#179]
**Describe the bug**
Array is pointing back to itself in an infinite loop.

**To Reproduce**
Run code:  
```VBA
Module Test
    Public Sub Main()
        Dim arr() As Variant
        arr = Array(Empty, arr)
        Stop
    End Sub
End Module
```
And when the code breaks go to the Locals windows and expand the array.

**Expected behavior**
In VBA we don't get this behaviour as a copy of the array is passed to the ```Array``` method.

**Screenshots**
![test](https://i.stack.imgur.com/rRZF5.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 Enterprise V1909
 - twinBASIC compiler v0.10.871

**Additional context**
None

**Comment by cristianbuse** [comment]
I have no idea if the behaviour in VBA is the same as in VB6 and I do not have VB6 to test it.

**Comment by WaynePhillipsEA** [comment]
Same in VB6.   Fixed in v0.10.933, thanks for reporting.
